tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -900,10 +901,14 @@ def test_log_metric(self):
         metric2 = entities.Metric(tkey, tval, get_current_time_millis() + 2, 0)
         nan_metric = entities.Metric("NaN", float("nan"), 0, 0)
         pos_inf_metric = entities.Metric("PosInf", float("inf"), 0, 0)
+        numpy_int_metric = entities.Metric("npInt64", int64(2), 0, 0)
         neg_inf_metric = entities.Metric("NegInf", -float("inf"), 0, 0)
         self.store.log_metric(run.info.run_id, metric)
         self.store.log_metric(run.info.run_id, metric2)
         self.store.log_metric(run.info.run_id, nan_metric)
+        # numpy type is tested because it's commonly returned from
+        # scikit-learn evaluation functions.
+        self.store.log_metric(run.info.run_id, numpy_int_metric)
```

Comment:
We'll need to update the assertion logic below (a new metric has been logged so the length of the logged metrics will increase)
Please also add a validation for the data being logged in the correct expected format within that validation block (lines 922:928).